dateparser/conf.py

```diff
@@ -21,7 +21,7 @@ class Settings(object):
     Default: False.
     * `PARSERS`: list of date parsers to use, in order of preference. Default:
     :attr:`dateparser.settings.default_parsers`.
-    """
+    """ # noqa E501
```

Comment:
Actually, I think this may be a good opportunity to improve the formatting of that docstring.